### PR TITLE
Readme: Add licensing terms

### PR DESCRIPTION
**Pull request by badcel**

### CODE REVIEW

File: logo.svg [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+     id="namedview66"
+     showgrid="false"
+     inkscape:zoom="0.3735005"
+     inkscape:cx="-239.44296"
+     inkscape:cy="-170.48603"
+     inkscape:window-x="1920"
+     inkscape:window-y="27"
+     inkscape:window-maximized="1"
+     inkscape:current-layer="layer1" />
+  <xhtml:script>(function(){function CaPHX() {
```
Comment (firox263) [inline logo.svg:50]: Don't know what's going on here, but seeing any kind of script inside an SVG makes me concerned. Looks like it's in my version too (Check the source of https://raw.githubusercontent.com/firox263/doc-test/master/logo.svg). It's most likely harmless, and part of the Inkscape SVG format (which has additions compared to standard SVG), but either way, code injection is dodgy.

I'll go ahead and push a plain-SVG version of the logo to this branch.

File: README.md [hunk 2]
```diff
@@ -70,3 +70,8 @@ The code in the library folder is not complete because the biggest part of the c
 [libchamplain]: https://wiki.gnome.org/Projects/libchamplain
 [GtkSharp]: https://github.com/GtkSharp/GtkSharp
 [sample_gtk_quickstart]: https://github.com/gircore/gir.core/tree/develop/Samples/Gtk3/Quickstart
+
+## Licensing terms
+Gir.Core is licensed under the terms of the MIT-License. Please see the [LICENSE](LICENSE) file for further information.
+
+The [Gir.Core logo](logo.svg) is build upon the work of the [original GTK logo](https://wiki.gnome.org/Projects/GTK/Logo) from Andreas Nilsson which is licensed under the [GNU Free Documentation License](https://www.gnu.org/licenses/fdl-1.3.txt) and was relicensed under CC BY-SA 3.0. Therefore the Gir.Core logo is licensed under the [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/deed.en), too.
```
Comment (firox263) [inline README.md:77]: Small typo. Should be "buil**t**".

**Comment by firox263** [comment]
@badcel This looks good to me. I've gone ahead and pushed a plain SVG version of the logo instead of the Inkscape one, which gets rid of the script tag. Can you have a look just to make sure I haven't missed anything? Otherwise, I think it's good to merge 👍

**Comment by badcel** [comment]
Thanks for fixing the file. I did not expect any scripts inside of it... Thanks for taking a real look into the file.

**Pull request merged**